### ISSUE TITLE
about tf.argmax() function

### ISSUE THREAD

**Issue by linrio** [naturomics/CapsNet-Tensorflow#6]
My tf **version is 1.2.1**. The following code in _capsNet.py_ :

`argmax_idx = tf.argmax(self.softmax_v, axis=1, output type=tf.int32)`

should be changed to:

`argmax_idx = tf.to_int32(tf.argmax(self.softmax_v, axis=1))` in version 1.2.1.

**Comment by naturomics** [comment]
Fixed.
